code/game/objects/items/storage/backpack.js

```diff
@@ -8,7 +9,10 @@ module.exports.templates = {
 				"StorageItem": {
 					max_size: 3,
 					max_combined_size: 21,
-					storage_slots: 21
+					storage_slots: 21,
+					populate_contents() {
```

Comment:
Don't do this.

Instead, go [here](https://github.com/Bluespess/tgstation-remake/blob/master/code/modules/jobs/job_types/job.js#L69) and set box to "survivalbox" (well if you follow my suggestion later about putting underscores "survival_box")